techniques/silverlight/SL13.html

```diff
@@ -211,7 +211,7 @@ <h3>Use SystemParameters.HighContrast to detect system high
     						infrastructure). Note that the added code calls a user-defined function
     						ChangeToHighCon(), which is the same function and behavior as shown
     						in Example 1 for the user-initiated high contrast switch. </p>
-            <pre xml:space="preserve">       public MainPage()
+            <pre>       public MainPage()
```

Comment:
```suggestion
            <pre><code>       public MainPage()
```
and add matching `</code>`